Source/WarpX.cpp

```diff
@@ -531,6 +532,11 @@ WarpX::ReadParameters ()
         pp.query("pml_has_particles", pml_has_particles);
         pp.query("do_pml_j_damping", do_pml_j_damping);
         pp.query("do_pml_in_domain", do_pml_in_domain);
+#ifdef WARPX_MAG_LLG
+        // Read the value of the time advancement scheme of M field
+        pp.query("mag_time_scheme_order", mag_time_scheme_order);
```

Comment:
Can you please add this as an input parameter in the sphinx documentation, and mention that the default is first order? This can be added in Docs/source/running_cpp/parameters.rst, in the section relevant to LLG

Reply:
I add it to the document. You can modify it if you want ~~